pages/pateter.tsx

```diff
@@ -97,6 +116,17 @@ const Pateter: NextPage<PateterProps> = ({allCommittees}) => {
           </ReactPageScroller>
         </div>
       </div>
+      
+      <div className="fixed flex flex-col items-center right-4 self-center top-1/4">
+          <NavBall index={0} scrollTo={() => scrollTo(0)} currentPage={currentPage} committeeyr={"Top"} ></NavBall>
```

Comment:
committeeyr? what does that mean? if it means 'year' then write committeeyear instead?